src/ddmd/expression.h

```diff
@@ -773,6 +773,8 @@ class DotIdExp : public UnaExp
 {
 public:
     Identifier *ident;
+    bool noderef;       // true if the result of the expression will never be dereferenced
```

Comment:
Good example of a confusing name - could be "no deref" or "node ref". `noDeref`?